search/src/Components/wrapper/reactive_search.js

```diff
@@ -559,11 +560,43 @@ class ReactiveSearchComponent extends Component {
         return (
             <ReactiveBase
                 app="weedid"
+                mapKey="AIzaSyDDiJ4QoRW9_DJEV94ehO3z8zfCHRuHfxk"
                 url={baseURL + "elasticsearch/"}
                 theme={theme}
                 headers={{'X-CSRFToken': Cookies.get('csrftoken')}}
             >
                 <div style={{ position: "fixed", width: "20rem", overflow: "scroll", height: "90%", left: 0, padding: '1rem' }}>
+                    <GeoDistanceSlider
+                      title="Location"
+                      componentId="geo_distance_filter"
+                      placeholder="Search Location"
+                      dataField="location"
+                      unit="km"
+                      URLParams
+                      showFilter={true}
+                      autoLocation={false}
+                      react={{
```

Comment:
Why are you not using `facetProps` here, or otherwise ensuring `react` is shared?